src/mcp/server/fastmcp/server.py

```diff
@@ -618,7 +618,7 @@ async def run_streamable_http_async(self) -> None:
         import uvicorn
 
         starlette_app = self.streamable_http_app()
-
+        starlette_app.router.redirect_slashes = False
```

Comment:
Hmmm... I think we can add `redirect_slashes` in the `Starlette` constructor on starlette's side.

But I think it should be okay.

Reply:
@Kludex So do we need to add this parameter to starlette now?